apps/condo/domains/meter/components/TabContent/PropertyMeter.tsx

```diff
@@ -0,0 +1,276 @@
+import { MetersImportWrapper } from '@condo/domains/meter/components/Import/Index'
+import { EXPORT_METER_READINGS_QUERY } from '@condo/domains/meter/gql'
+import {
+    PropertyMeter,
+    METER_TAB_TYPES,
+    MeterReadingFilterTemplate,
+} from '@condo/domains/meter/utils/clientSchema'
+
+
+const METERS_PAGE_CONTENT_ROW_GUTTERS: RowProps['gutter'] = [0, 40]
+const FILTERS_CONTAINER_GUTTER: RowProps['gutter'] = [16, 16]
+const RESET_FILTERS_BUTTON_STYLE: CSSProperties = { paddingLeft: 0 }
+const QUICK_FILTERS_COL_STYLE: CSSProperties = { alignSelf: 'center' }
+
+const SORTABLE_PROPERTIES = ['date', 'clientName', 'source']
+
+type PropertyMetersTableContentProps = {
+    filtersMeta: FiltersMeta<MeterReadingWhereInput>[]
+    tableColumns: ColumnsType
+    baseSearchQuery: MeterReadingWhereInput
+    canManageMeters: boolean
+    sortableProperties?: string[]
+    mutationErrorsToMessages?: Record<string, string>
+    loading?: boolean
+}
+
+const PropertyMetersTableContent: React.FC<PropertyMetersTableContentProps> = ({
+    filtersMeta,
+    tableColumns,
+    baseSearchQuery,
+    canManageMeters,
+    sortableProperties,
+    loading,
+}) => {
+    const intl = useIntl()
+    const CreateMeterButtonLabel = intl.formatMessage({ id: 'pages.condo.meter.index.CreateMeterButtonLabel' })
+    const SearchPlaceholder = intl.formatMessage({ id: 'filters.FullSearch' })
+    const IsActiveMessage = intl.formatMessage({ id: 'pages.condo.meter.Meter.isActive' })
+    const OutOfOrderMessage = intl.formatMessage({ id: 'pages.condo.meter.Meter.outOfOrder' })
+
+    const router = useRouter()
+    const { filters, offset, sorters, tab } = parseQuery(router.query)
+    const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
+
+    const [isShowActiveMeters, setIsShowActiveMeters] = useState(true)
+    const [isShowArchivedMeters, setIsShowArchivedMeters] = useState(false)
+
+    const { filtersToWhere, sortersToSortBy } = useQueryMappers(filtersMeta, sortableProperties || SORTABLE_PROPERTIES)
+
+    const sortBy = useMemo(() => sortersToSortBy(sorters) as SortPropertyMetersBy[], [sorters, sortersToSortBy])
+
+    const searchPropertyMetersQuery = useMemo(() => ({
+        ...filtersToWhere(filters),
+        ...(isShowActiveMeters && !isShowArchivedMeters)  ? { archiveDate: null } : {},
+        ...(isShowArchivedMeters && !isShowActiveMeters) ? { archiveDate_not: null } : {},
+        ...baseSearchQuery,
+    }), [baseSearchQuery, filters, filtersToWhere, isShowActiveMeters, isShowArchivedMeters])
+
+    const {
+        loading: propertyMetersLoading,
+        count: total,
+        objs: propertyMeters,
+        refetch,
+    } = PropertyMeter.useObjects({
+        sortBy,
+        where: searchPropertyMetersQuery,
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    }, {
+        fetchPolicy: 'network-only',
```

Comment:
Why do you need to fetch it by network every time? It might be too slow in some cases